### ISSUE TITLE
command not recongize error

### ISSUE THREAD

**Issue by EasonWang01** [andris9/smtp-server#72]
I was using the examples/server.js  

and using nodemailer to connect

but some error occur

```
[2016-11-20 10:38:09] INFO: SMTP Server listening on [::]:2525
[2016-11-20 10:39:02] INFO: [wMTJj9/pXmX7] Connection from [127.0.0.1]
[2016-11-20 10:39:02] DEBUG: [wMTJj9/pXmX7] S: 220 easonwang-mbp.local ESMTP Welcome to My Awesome SMTP Server
[2016-11-20 10:39:03] DEBUG: [wMTJj9/pXmX7] C: EHLO easonwang-mbp.local
[2016-11-20 10:39:03] DEBUG: [wMTJj9/pXmX7] S: 250-easonwang-mbp.local Nice to meet you, [127.0.0.1]
250-8BITMIME
250-SMTPUTF8
250-SIZE 10485760
250-XCLIENT NAME ADDR PORT PROTO HELO LOGIN
250 XFORWARD NAME ADDR PORT PROTO HELO IDENT SOURCE
[2016-11-20 10:39:03] DEBUG: [wMTJj9/pXmX7] C: AUTH PLAIN AGphc29uNzEyNDgAMjkxMTQwNTU=
[2016-11-20 10:39:03] DEBUG: [wMTJj9/pXmX7] S: 500 Error: command not recognized
[2016-11-20 10:39:03] INFO: [wMTJj9/pXmX7] Connection closed to [127.0.0.1]
```

**Comment by pranshu792** [comment]
@EasonWang01 Can you please send the AUTH event handler of SMTP server ?

**Comment by pranshu792** [comment]
@EasonWang01 As stated by you, this issue has been resolved somehow by you. Could you please close this ?

**Comment by vanguard1127** [comment]
Hello!
I tested your server on windows server.
But I met the error like this.

[2019-12-13 17:44:51] INFO  [#65thhdttqvjjezv7] Connection from group-ib.com
[2019-12-13 17:44:51] DEBUG [#65thhdttqvjjezv7] S: 220 EC2AMAZ-QQOUVHP ESMTP Welcome to My Awesome SMTP Server
[2019-12-13 17:44:51] DEBUG [#65thhdttqvjjezv7] C: AUTH TLS
[2019-12-13 17:44:51] DEBUG [#65thhdttqvjjezv7] S: 500 Error: command not recognized
[2019-12-13 17:44:51] INFO  [#65thhdttqvjjezv7] Connection closed to group-ib.com
[2019-12-14 00:54:39] DEBUG [#gfrxer23uqg3dj35] C: EHLO zg-0911b-41
[2019-12-14 00:54:39] DEBUG [#gfrxer23uqg3dj35] S: 421 EC2AMAZ-QQOUVHP You talk too soon
[2019-12-14 00:54:39] INFO  [#gfrxer23uqg3dj35] Connection closed to 159.203.201.23

In this case , I send emails to my server with using https://www.smtper.net/.

Can you please send solution?
